mod/likes/views/default/likes/popup.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+$guid = get_input("guid");
+
+elgg_entity_gatekeeper($guid);
```

Comment:
What happens if likes is requested and this is triggered? What shows up in the lightbox?